pkg/cmd/util/clientcmd/factory.go

```diff
@@ -510,12 +510,12 @@ func NewFactory(clientConfig kclientcmd.ClientConfig) *Factory {
 			if t.Spec.Paused {
 				return true, nil
 			}
-			t.Spec.Paused = true
+			body := []byte("{\"spec\":{\"paused\":true}}")
```

Comment:
> @deads2k I think CalculatePatches is an overkill for update simple like this, wdyt?

I could see creating a more focused helper.  If you want this to become "standard", then it needs to be easy and maintainable.  I think that bugs caused by typos in string serialized json strategic merge patches will be nearly impossible to stamp out.